### DISCUSSION TITLE
The new feed appears to display less information?

### DISCUSSION THREAD

**Discussion by olliefr** [community/community#13146] [Other Feature Feedback, Questions, & Ideas]
I appreciate the use of vertical space for clarity, but it appears to be less informative than the old feed because there's less information being displayed in one screen's worth of space.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by olliefr** [reply]
This is great, but my comment was on a different issue. The generous use of vertical space in the new "for you" feed would require me to scroll for much longer to consume the same amount of information as what the traditional "following" feed would display. The practical outcome is that I don't look at the new feed because the information density is so low. It's especially obvious on laptop screens which tend not to have much vertical space. The new feed looks goods, but is not very practical, IMO.
